docs/rpc-spec.md

```diff
@@ -1000,3 +1002,8 @@ Transmission 4.0.0 (`rpc-version-semver` 5.3.0, `rpc-version`: 17)
 | `group-set` | new method
 | `group-get` | new method
 
+Transmission 4.0.1 (`rpc-version-semver` 5.4.0, `rpc-version`: 18)
```

Comment:
Since this PR is an enhancement rather than a bugfix, this is a semver-minor change and needs to be `4.1.0` rather than `4.0.1`.

The rest of the PR LGTM

```suggestion
Transmission 4.1.0 (`rpc-version-semver` 5.4.0, `rpc-version`: 18)
```